test/libsolidity/semanticTests/inheritance/transient_storage_state_variable.sol

```diff
@@ -0,0 +1,22 @@
+contract A {
+    uint transient x;
+    int y;
+}
+
+contract C is A {
+    uint w;
+    int transient z;
```

Comment:
What about smaller types so that we get more non-zero offsets? I see you added one new test for that, but I'd suggest just making it more varied in other tests, so that this gets more incidental coverage. Including this test in particular.